src/NodeAutoCompleteViewExtension/ViewModels/NodeAutoCompleteBarViewModel.cs

```diff
@@ -695,6 +642,16 @@ internal MLNodeClusterAutoCompletionResponse GetMLNodeClusterAutocompleteResults
             return results;
         }
 
+        private MLNodeAutoCompletionResponse GetMLNodeAutocompleteResults()
```

Comment:
do we really need these extra methods ? 
we can perhaps simplify and just use something like GetAutocompleteResult<T>(string req) directly ?